packages/react/src/primitives/Collection/Collection.tsx

```diff
@@ -65,7 +65,7 @@ export const Collection = <Item,>({
 
   // Pagination
   const pagination = usePagination({
-    totalPages: Math.floor(items.length / itemsPerPage),
+    totalPages: getPageCount(items.length, itemsPerPage),
```

Comment:
This is such a great example of a time when abstracting to another function for unit testing is extremely valuable!